integration-tests/container-image/pom.xml

```diff
@@ -7,91 +7,13 @@
     <artifactId>quarkus-integration-tests-parent</artifactId>
     <version>999-SNAPSHOT</version>
   </parent>
-  <artifactId>quarkus-integration-test-container-image</artifactId>
-  <name>Quarkus - Integration Tests - Container Image</name>
-  <properties>
-    <skipTests>true</skipTests>
-  </properties>
-  <dependencyManagement>
-    <dependencies>
-      <dependency>
-        <groupId>com.google.j2objc</groupId>
-        <artifactId>j2objc-annotations</artifactId>
-        <version>1.3</version>
-      </dependency>
-    </dependencies>
-  </dependencyManagement>
-  <dependencies>
-    <dependency>
-      <groupId>io.quarkus</groupId>
-      <artifactId>quarkus-core-deployment</artifactId>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>io.quarkus</groupId>
-      <artifactId>quarkus-test-maven</artifactId>
-      <scope>test</scope>
-    </dependency>
-  </dependencies>
-  <build>
-    <testResources>
-      <testResource>
-        <directory>src/it</directory>
-        <filtering>true</filtering>
-      </testResource>
-    </testResources>
-    <plugins>
-      <plugin>
-        <artifactId>maven-invoker-plugin</artifactId>
-        <version>3.1.0</version>
-        <executions>
-          <execution>
-            <id>integration-tests</id>
-            <goals>
-              <goal>install</goal>
-              <goal>run</goal>
-              <goal>verify</goal>
-            </goals>
-          </execution>
-        </executions>
-        <configuration>
-          <cloneProjectsTo>${project.build.directory}/it</cloneProjectsTo>
-          <cloneClean>true</cloneClean>
-          <postBuildHookScript>verify</postBuildHookScript>
-          <addTestClassPath>true</addTestClassPath>
-          <skipInvocation>${skipTests}</skipInvocation>
-          <streamLogs>true</streamLogs>
-          <invokerPropertiesFile>invoker.properties</invokerPropertiesFile>
-        </configuration>
-      </plugin>
-      <plugin>
-        <artifactId>maven-failsafe-plugin</artifactId>
-        <executions>
-          <execution>
-            <goals>
-              <goal>integration-test</goal>
-              <goal>verify</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
-    </plugins>
-  </build>
 
-  <profiles>
-    <profile>
-      <id>container-image-docker</id>
-      <activation>
-        <property>
-          <name>docker</name>
-        </property>
-        <os>
-          <family>linux</family>
-        </os>
-      </activation>
-      <properties>
-        <skipTests>false</skipTests>
-      </properties>
-    </profile>
-  </profiles>
+  <artifactId>quarkus-integration-test-container-image-parent</artifactId>
+  <name>Quarkus - Integration Tests - Container Image - Parent</name>
+  <packaging>pom</packaging>
+
+  <modules>
+    <module>invoker</module>
+    <module>standard</module>
```

Comment:
I don't like `standard` name. Can be more descriptive name picked?

Reply:
Do you have a suggestion? I suck horribly at naming.

FWIW we already use that name in kubernetes tests.

Reply:
so there is maven-invoker way and standard Quarkus way of working with container-image

what about:
 - invoker => maven-invoker-way
 - standard => quarkus-standard-way
?

Reply:
I don't particularly like `way` in there TBH. @gsmet any good suggestions here?

Reply:
I couldn't come up with anything better, so I went with your suggestion.